### PR TITLE
beta-release-5.6.0 notes

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

**Comment by ScottSuarez** [comment]
cherry-picked https://github.com/hashicorp/terraform-provider-google-beta/commit/d5852061e1faaa9ea16f693cb44c43b17d456b85 to fix `TestAccSqlDatabaseInstance_basicMSSQL`
https://github.com/hashicorp/terraform-provider-google-beta/commits/main

**Comment by ScottSuarez** [comment]
Cherry picked revert of 90ca686673a117770265bf9cc828edff290b80f3 - https://github.com/hashicorp/terraform-provider-google-beta/pull/6628
https://github.com/hashicorp/terraform-provider-google-beta/commits/release-5.6.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,32 @@
 ## 5.6.0 (Unreleased)
+FEATURES:
+* **New Data Source:** `google_backup_dr_management_server` ([#6596](https://github.com/hashicorp/terraform-provider-google-beta/pull/6596))
+* **New Resource:** `google_compute_instance_settings` ([#6615](https://github.com/hashicorp/terraform-provider-google-beta/pull/6615))
+* **New Resource:** ``google_integration_connectors_connection`([#6612](https://github.com/hashicorp/terraform-provider-google-beta/pull/6612))
```
Comment (slevenick) [inline CHANGELOG.md:5]:
```suggestion
* **New Resource:** `google_integration_connectors_connection`([#6612](https://github.com/hashicorp/terraform-provider-google-beta/pull/6612))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,32 @@
...
+* monitoring: added `pings_count`, `user_labels`, and `custom_content_type` fields to `google_monitoring_uptime_check_config` resource ([#6594](https://github.com/hashicorp/terraform-provider-google-beta/pull/6594))
+* spanner: added `autoscaling_config` field to  `google_spanner_instance` ([#6616](https://github.com/hashicorp/terraform-provider-google-beta/pull/6616))
+* sql: update `google_sql_database_instance` to reflect that `data_cache_enabled` works for PostgresQL now, in addition to MYSQL ([#6595](https://github.com/hashicorp/terraform-provider-google-beta/pull/6595))
+* sql: added `ssl_mode` field to `google_sql_database_instance` resource ([#6579](https://github.com/hashicorp/terraform-provider-google-beta/pull/6579))
+* vertexai: added `private_service_connect_config` to `google_vertex_ai_index_endpoint` ([#6614](https://github.com/hashicorp/terraform-provider-google-beta/pull/6614))
+* workstations: add `domain_config` field to resource `google_workstations_workstation_cluster` (beta) ([#6609](https://github.com/hashicorp/terraform-provider-google-beta/pull/6609))
```
Comment (slevenick) [inline CHANGELOG.md:26]:
```suggestion
* workstations: added `domain_config` field to resource `google_workstations_workstation_cluster`([#6609](https://github.com/hashicorp/terraform-provider-google-beta/pull/6609))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,32 @@
...
+* gkehub: added `metrics_gcp_service_account_email` to `google_gke_hub_feature_membership` ([#6597](https://github.com/hashicorp/terraform-provider-google-beta/pull/6597))
+* logging: added `index_configs` field to `logging_bucket_config` resource ([#6598](https://github.com/hashicorp/terraform-provider-google-beta/pull/6598))
+* logging: added `index_configs` field to `logging_project_bucket_config` resource ([#6598](https://github.com/hashicorp/terraform-provider-google-beta/pull/6598))
+* monitoring: added `pings_count`, `user_labels`, and `custom_content_type` fields to `google_monitoring_uptime_check_config` resource ([#6594](https://github.com/hashicorp/terraform-provider-google-beta/pull/6594))
+* spanner: added `autoscaling_config` field to  `google_spanner_instance` ([#6616](https://github.com/hashicorp/terraform-provider-google-beta/pull/6616))
+* sql: update `google_sql_database_instance` to reflect that `data_cache_enabled` works for PostgresQL now, in addition to MYSQL ([#6595](https://github.com/hashicorp/terraform-provider-google-beta/pull/6595))
```
Comment (slevenick) [inline CHANGELOG.md:23]:
```suggestion
* sql: updated `google_sql_database_instance` to reflect that `data_cache_enabled` works for PostgresQL now, in addition to MYSQL ([#6595](https://github.com/hashicorp/terraform-provider-google-beta/pull/6595))
```

Comment (slevenick) [reply CHANGELOG.md:23]: Actually, this is docs-only so should be removed?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,32 @@
...
+* **New Resource:** `google_compute_instance_settings` ([#6615](https://github.com/hashicorp/terraform-provider-google-beta/pull/6615))
+* **New Resource:** ``google_integration_connectors_connection`([#6612](https://github.com/hashicorp/terraform-provider-google-beta/pull/6612))
+
+IMPROVEMENTS:
+* assuredworkloads: added `enable_sovereign_controls`, `partner`, `partner_permissions`, `violation_notifications_enabled`, and several other output-only fields to `google_assured_workloads_workloads` ([#6597](https://github.com/hashicorp/terraform-provider-google-beta/pull/6597))
+* composer: added `storage_config` to `google_composer_environment` ([#6606](https://github.com/hashicorp/terraform-provider-google-beta/pull/6606))
+* container: added `fleet` field to `google_container_cluster` resource ([#6610](https://github.com/hashicorp/terraform-provider-google-beta/pull/6610))
+* container: marked `binary_authorization.evaluation_mode` field as computed and optional in `google_container_cluster` resource ([#6610](https://github.com/hashicorp/terraform-provider-google-beta/pull/6610))
```
Comment (slevenick) [inline CHANGELOG.md:11]: What does this mean for users?

Comment (ScottSuarez) [reply CHANGELOG.md:11]: not much, going to remove. realistically it means that this can now support a computed value

Comment (ScottSuarez) [reply CHANGELOG.md:11]: Needed due to fleet, but fleet is covered above. I don't think the impact is really necessary detail

**Review by slevenick — APPROVED** [review]

**Pull request merged**